scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/GraphQLMerlinService.java

```diff
@@ -596,11 +624,8 @@ public MissionModelTypes getMissionModelTypes(final PlanId planId)
         }
         """.formatted(planId.id());
     final JsonObject response;
-    try {
-      response = postRequest(request).get();
-    } catch (PlanServiceException e) {
-      throw new MissionModelServiceException("Failed to get mission model types for plan id %s".formatted(planId), e);
-    }
+    response = postRequest(request).get();
+
```

Comment:
Why is this try-catch no longer needed? Same question on L700 and L723

Reply:
because `MissionModelServiceException` does not exist anymore.